### PR TITLE
Various i18n fixes/optimizations

### LINKED ISSUE #106: Translation: Combine strings even if context is different
In the functions.php your are using exactly the same string but in 2 different contexts. This means that translators have to translate exactly the same string twice. To be honest I think the context here is irrelevant for any language, so proposal is to remove the context.
image

### LINKED ISSUE #108: Translation: Consistency in wording
image

Try to keep the same wording, so use 'auto' or 'automatic' everywhere but don't mix.

### PR DESCRIPTION
**Pull request by audrasjb**
In this PR:
- Remove useless/redundant `aria-label` atttributes
Fixes #106 and partially #94 (see [this comment](https://github.com/WordPress/wp-autoupdates/issues/94#issuecomment-622570197))
- Replace several `automatic updates` with `auto-updates` for better consistency
Fixes #108 

I kept some of the `automatic updates` strings, when they appear in error messages.

### CODE REVIEW

**Review by pbiron — COMMENTED** [review]
I think it will be easier if these changes are made in #103 instead.

Because if this is merged, then `master` won't be able to merged into that PR branch without resolving conflicts because that PR changes all (or at least most) of the same lines for other reasons.

**Comment by audrasjb** [comment]
Ok, so let's say that we wait for #103 to be merged and then I refreshes this PR 🙂

**Comment by pbiron** [comment]
> Ok, so let's say that we wait for #103 to be merged and then I refreshes this PR 🙂

That's probably going to cause similar git conflicts.  So, maybe close this PR, what until #103 is merged and then open a new PR?

**Comment by azaozz** [comment]
Right, would be easier to merge 103 then refresh and merge these changes.

As far as I see all the changes here look good.

**Comment by audrasjb** [comment]
@pbiron I updated the PR accordingly, could you please take a look?

**Review by pbiron — APPROVED** [review]
Looks great!

**Pull request merged**
